src/query/query_parser/query_parser.rs

```diff
@@ -506,6 +507,13 @@ impl QueryParser {
                 let bytes_term = Term::from_field_bytes(field, &bytes);
                 Ok(vec![LogicalLiteral::Term(bytes_term)])
             }
+            FieldType::Ip(ref option) => {
+                if !option.is_indexed() {
+                    return Err(QueryParserError::FieldNotIndexed(field_name.to_string()));
+                }
+                let text_term = Term::from_field_text(field, phrase);
```

Comment:
I'm ok with using text for IP if you do it right, but this here is not good at all.
A given IP have way more than a single representation.

Reply:
I'll remove it from this PR, we can add it later